### ISSUE TITLE
Views : Aggregation not taken in account

### ISSUE THREAD

**Issue by erwinvanhunen** [SharePoint/PnP-Sites-Core#924]
_From @Amysno on December 1, 2016 16:39_

There is a bug when creating views : If we want to aggregate some prices, it does not work.

Here is my View declared on the ListsCreation template. This view is supposed to show elements grouped by Lot and then ordered by Order. We want a Sum on montantecheance, which is a currency field. Here is the template :
 ```
<View Name="{4A2FDED4-0E32-4494-A586-B845116689F4}" DefaultView="TRUE" MobileView="TRUE" Type="HTML" DisplayName="Suivi de facturation" Url="/teams/NAN/NA16060004/Lists/Suivi de facturation/Suivi de facturation.aspx" Level="1" BaseViewID="1" ContentTypeID="0x" ImageUrl="/_layouts/15/images/generic.png?rev=44">
              <Query>
                <GroupBy Collapse="FALSE" GroupLimit="30">
                  <FieldRef Name="lot" />
                </GroupBy>
                <OrderBy>
                  <FieldRef Name="ordre" />
                </OrderBy>
              </Query>
              <ViewFields>
                <FieldRef Name="numcommande" />
                <FieldRef Name="LinkTitle" />
                <FieldRef Name="pourcentecheance" />
                <FieldRef Name="montantecheance" />
                <FieldRef Name="dateprevisionnelle" />
                <FieldRef Name="datefacturation" />
                <FieldRef Name="etatfactu" />
              </ViewFields>
              <RowLimit Paged="TRUE">30</RowLimit>
              <Aggregations Value="On">
                <FieldRef Name="montantecheance" Type="SUM" />
              </Aggregations>
              <JSLink>clienttemplates.js</JSLink>
            </View>
```

The group and order by are working but not the aggregation.
Or maybe you could tell me if i am doing something wrong here.

Thanks and have a nice day

_Copied from original issue: SharePoint/PnP-Provisioning-Schema#142_

**Comment by erwinvanhunen** [comment]
_From @PaoloPia on December 2, 2016 16:37_

Hi,
Thanks for your feedback. So far we do not support aggregations. I created an issue (https://github.com/SharePoint/PnP-Sites-Core/issues/876) in the PnP-Sites-Core repository, which is the right one for such kind of requests. We will follow up on this request there. I will close this item. Thanks.

**Comment by erwinvanhunen** [comment]
This has been fixed now with PR #926 . I will close this issue.
